packages/static/kairos-overlay-files/files/etc/cos/bootargs.cfg

```diff
@@ -1,5 +1,5 @@
 function setSelinux {
-    source (loop0)/etc/os-release
+    . (loop0)/etc/os-release
```

Comment:
This is executed by the grub shell so I don't think it's affected by this and in fact might break things if changed as the source command exists but the fot may not

Reply:
correct, this shouldn't be part of the changeset